### PR TITLE
Added article about official MCR endpoints

### PR DESCRIPTION
**Pull request by toddysm**

### CODE REVIEW

**Review by SteveLasker — APPROVED** [review]
LGTM
Only additional suggestion is to add a link in the readme so people can find the content

File: docs/mcr-endpoints-guidance.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ms.topic: guidance
+ms.date: 02/03/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Guidance for using the official Microsoft Container Registry (MCR) endpoints
+
+Microsoft Container Registry (MCR) is an implementation of the [OCI Distribution Specification][oci-spec] which delivers [artifacts][oci-artifacts], such as container images, helm charts and other assets distributed by Microsoft and Azure. The Distribution Spec linked above defines two endpoints:
```
Comment (bariscaglar) [inline docs/mcr-endpoints-guidance.md:13]: Not sure about Microsoft _and_ Azure. Makes it read like they are two separate entities. Maybe Microsoft including Azure?

File: docs/mcr-endpoints-guidance.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- **Registry Endpoint**: providing content discovery. This is also referred to as *REST Endpoint* and it is the endpoint users are most familiar with when pulling an image; for example: `docker pull mcr.microsoft.com/windows/servercore:1909`. The registry endpoint is load balanced across multiple worldwide regions, providing reliable and consistent access to the MCR artifacts catalog.
```
Comment (bariscaglar) [inline docs/mcr-endpoints-guidance.md:15]: **P**roviding (same as below).

File: docs/mcr-endpoints-guidance.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- **Data Endpoint**: providing content delivery. As a registry client discovers the content required, it negotiates a set of content urls, pulling from the data endpoint. To provide reliable delivery of content, the data endpoint is backed by regional CDN endpoints.
```
Comment (bariscaglar) [inline docs/mcr-endpoints-guidance.md:16]: URLs should be captalized

File: docs/mcr-endpoints-guidance.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+To access MCR, the following FQDNs should be used.
+
+| Purpose | Protocol | Target FQDN |
+| - | - | - |
+| Registry Endpoint | https | `mcr.microsoft.com` |
+| Data Endpoint | https | `*.data.mcr.microsoft.com` |
+
+> **Note:** MCR provides global coverage through Azure Traffic Manager for the registry endpoint, with regional CDNs managed by Azure Front Door for the data endpoints.
+> Over time, MCR team will continue adding regional endpoints to ensure the best performance for our customers. Check the [MCR Firewall Rules][mcr-firewall-rules] article for details how to configure your firewall rules.
```
Comment (bariscaglar) [inline docs/mcr-endpoints-guidance.md:28]: This link didn't work for me. Is it supposed to work after the article is published?

**Review by bariscaglar — CHANGES_REQUESTED** [review]

File: docs/mcr-endpoints-guidance.md (hunk 1 above)
Comment (toddysm) [reply docs/mcr-endpoints-guidance.md:13]: Removed Azure.

File: docs/mcr-endpoints-guidance.md (hunk 2 above)
Comment (toddysm) [reply docs/mcr-endpoints-guidance.md:15]: Fixed

File: docs/mcr-endpoints-guidance.md (hunk 4 above)
Comment (toddysm) [reply docs/mcr-endpoints-guidance.md:28]: Fixed

File: docs/mcr-endpoints-guidance.md (hunk 3 above)
Comment (toddysm) [reply docs/mcr-endpoints-guidance.md:16]: FIxed

**Review by bariscaglar — APPROVED** [review]

**Pull request merged**